Essentials/src/com/earth2me/essentials/Mob.java

```diff
@@ -92,8 +92,8 @@
     WANDERING_TRADER("WanderingTrader", Enemies.FRIENDLY, "WANDERING_TRADER"),
     BEE("Bee", Enemies.NEUTRAL, "BEE"),
     STRAY("Stray", Enemies.ENEMY, "STRAY"),
-    HOGLIN("Hoglin", Enemies.ENEMY, "HOGLIN"),
-    PIGLIN("Piglin", Enemies.NEUTRAL, "PIGLIN"),
+    HOGLIN("Hoglin", Enemies.ADULT_ENEMY, "HOGLIN"),
+    PIGLIN("Piglin", Enemies.ADULT_ENEMY, "PIGLIN"),
```

Comment:
Shouldn't this just be enemy? Not sure why this needs a new enum. (btw piglin is apparently hostile by default, so it should probably be set as enemy, my bad)
```suggestion
    HOGLIN("Hoglin", Enemies.ENEMY, "HOGLIN"),
    PIGLIN("Piglin", Enemies.ENEMY, "PIGLIN"),
```